### DISCUSSION TITLE
Assignes -- can't add more than 1 assignes to project

### DISCUSSION THREAD

**Discussion by billpebert** [community/community#19800] [Projects and Issues]
Why i can't add other account/team member to the issues?
<img width="187" alt="image" src="https://user-images.githubusercontent.com/70867121/176848048-5a03506c-7727-4692-97b8-9e67ff568ac5.png">

**Comment by Zi-Tao** [comment]
@billpebert  https://superuser.com/questions/1543801/why-does-github-limit-the-number-of-people-that-can-be-assigned-to-a-task-in-cer
